src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
According to the checklist, interface names should not be verbs. Consider renaming this interface to something that represents its purpose more accurately, such as 'Drawable' or 'DrawOperation'.